docs/pages/versions/v32.0.0/guides/using-firebase.md

```diff
@@ -8,7 +8,7 @@ Luckily, the Firebase JavaScript SDK starting from version 3.1+ has almost full
 
 See [firebase.google.com/docs/database](https://firebase.google.com/docs/database) for more general information and the [official Firebase blog post announcing React Native compatibility](https://firebase.googleblog.com/2016/07/firebase-react-native.html)
 
-> **Note:** This guide only covers Firebase Realtime Database, and not the other services under the larger Google Firebase umbrella. Firebase Cloud Storage is currently not supported, but we are [working on upstreaming a Blob implementation](https://github.com/facebook/react-native/issues/11103) to React Native that would make this possible. For more background on why other Firebase services are not supported, please read [Brent Vatne's response on Canny](https://expo.canny.io/feature-requests/p/full-native-firebase-integration)
+> **Note:** This guide mostly covers Firebase Realtime Database (and some Firestore as well), but not the other services under the larger Google Firebase umbrella. Firebase Cloud Storage is currently not supported, but we are [working on upstreaming a Blob implementation](https://github.com/facebook/react-native/issues/11103) to React Native that would make this possible. For more background on why other Firebase services are not supported, please read [Brent Vatne's response on Canny](https://expo.canny.io/feature-requests/p/full-native-firebase-integration)
```

Comment:
could you update this, storage has been possible for some time now

Reply:
Sure, I will try, although I've never used it myself (I've only seriously been using Expo for a few weeks now).  Looks like this is the package: https://www.npmjs.com/package/expo-firebase-storage ... yes?

Reply:
nope you can use web storage with Expo. That library is used for native firebase storage.

Reply:
https://github.com/EvanBacon/firebase-instagram/blob/master/utils/uploadPhoto.js

Reply:
As I stated in my commit, I added text that I thought would be helpful to future "new people" (myself included) who are just starting to use Expo with Firebase (and might be having trouble navigating the docs and code).